lib/rdr/configurable.rb

```diff
@@ -16,7 +21,7 @@ module Configurable
 
       # Contact email for export files
       mattr_accessor :export_files_contact_email do
-        ENV["EXPORT_FILES_CONTACT_EMAIL"] || 'ddrhelp@duke.edu'
+        ENV["EXPORT_FILES_CONTACT_EMAIL"] || 'repository@duke.edu'
```

Comment:
If this is not a real address, I would default to something that like "no-reply@duke.edu".